auslib/blobs/apprelease.py

```diff
@@ -75,13 +75,22 @@ def getBuildID(self, platform, locale):
                 raise BadDataError("No buildID for platform '%s'" % (platform))
             return self['platforms'][platform]['buildID']
 
+    @property
+    def has_wnp(self):
+        # check for What's new page for Blobs < v9
+        for group in self.get('openURL', []):
```

Comment:
This doesn't appear to be working: https://screenshots.firefox.com/BsG3coeqJOFaphwl/localhost -- most likely because openURL is just a string for <V9, not a list of dicts.

Reply:
Could you please send me an example of <V9 blob to test with?

Reply:
You should have `Firefox-52.0.1esr-build2-WNP` in your local database -- that's a V4 blob with a WNP. `Firefox-57.0.1-build2` is a V4 blob without a WNP.

Reply:
Thanks 👍